<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="ROIImageGraphicsView" name="ROIgraphicsView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>581</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BoxROI_button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Box ROI</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SinglePt_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Single Pt</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="multiPt_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Multi Pt</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="polygon_button">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Polygon</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="free_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Free</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_polygon_button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>570</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Poly</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_last_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>600</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Last</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_all_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>630</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete All</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="multiPt_new_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>570</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="multiPt_move_button">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>570</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_free_button">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>570</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Free</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ROIImageGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>ROIImageGraphicsView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
